server/game/cards/events/01/risenfromthesea.js

```diff
@@ -14,6 +14,7 @@ class RisenFromTheSea extends DrawCard {
             handler: context => {
                 context.event.saveCard(context.target);
                 this.controller.attach(this.controller, this, context.target, 'play');
+                this.setCardType('attachment');
```

Comment:
If a Nightmares-style blanking card for attachments would be printed, it would not blank the Condition trait, which I guess would lead to a bug then in this case.

On the other hand, we have seen a trait blanker as well (The White Shadows), so speculation on that front seems a bit pointless I think.